src/OrchardCore.Modules/OrchardCore.OpenId/Manifest.cs

```diff
@@ -52,9 +52,9 @@
 
 [assembly: Feature(
     Id = OpenIdConstants.Features.Validation,
-    Name = "OpenID Token Validation",
+    Name = "OpenID Connect Token Validation",
+    Description = "Validates tokens issued by the OpenID Connect Authorization Server or other trusted servers using JWT and OpenID Connect discovery.",
```

Comment:
IMO, `supporting` rather than `using` was clearer here.

```suggestion
    Description = "Validates tokens issued by the local OpenID Connect authorization server or other trusted servers supporting JWT and OpenID Connect discovery.",
```